src/google_apps_clj/google_sheets_v4.clj

```diff
@@ -64,8 +65,34 @@
       (.get spreadsheet-id)
       (.execute)))
 
+(defn get-sheet-titles
+  [service spreadsheet-id]
+  "returns a list of [sheet-title sheet-id] tuples. It seems the order reflects
+  the order of the tabs in google's interface, though I doubt this is anywhere
+  guaranteed."
+  (->> (get (get-sheet-info service spreadsheet-id) "sheets")
+       (map #(get % "properties"))
+       (map (juxt #(get % "title") #(get % "sheetId")))))
```

Comment:
Also I wonder if there's any utility in exploring `Spreadsheet` and `Sheet` records? For the latter, that could be a nice place to hang Comparable.